src/chainconsumer/chain.py

```diff
@@ -395,40 +397,52 @@ def from_numpyro(
         cls,
         mcmc: numpyro.infer.MCMC,
         name: str,
+        var_names: list[str] = [],
```

Comment:
We should probably make this `list[str] | None = None` or similar to ensure we dont have mutable kwarg defaults